daemon/src/model.rs

```diff
@@ -109,7 +110,17 @@ pub struct WalletInfo {
 }
 
 #[derive(
-    Debug, Clone, Copy, SerializeDisplay, DeserializeFromStr, PartialEq, Eq, Hash, PartialOrd, Ord,
+    Debug,
+    Clone,
+    Copy,
+    SerializeDisplay,
+    DeserializeFromStr,
+    PartialEq,
+    Eq,
+    Hash,
+    PartialOrd,
+    Ord,
+    sqlx::Type,
```

Comment:
Adding this without `transparent` or another annotation does not do anything on structs as far as I know.